source/documentation/administration_guide/topics/Renaming_the_Manager_with_the_Ovirt_Engine_Rename_Tool.adoc

```diff
@@ -55,3 +55,28 @@ This command modifies the FQDN in each self-hosted engine node's local copy of [
 This command modifies the FQDN in the main copy of [filename]`/etc/ovirt-hosted-engine-ha/hosted-engine.conf` on the shared storage domain.
 
 Now, all new and existing self-hosted engine nodes use the new FQDN.
+
+
+[NOTE]
+====
+The oVirt Engine Rename Tool is designed to work only on local machines. Changing the engine name does not automatically update the name on remote Data Warehouse machines. Changing the names on remote DWH machines must be performed manually.
+====
+
+*For remote Data Warehouse deployments, follow these steps on the remote machine:*
```

Comment:
I want to emphasize that the steps will not be done on the engine machine, so:
Add a line below the sentence: follow these steps on the remote machine,
Or add 'Not on the engine machine'